### DISCUSSION TITLE
Open-webui API: create/enable/disable users?

### DISCUSSION THREAD

**Discussion by stevenbaert** [open-webui/open-webui#7049] [Q&A]
Hi,

I found this documentation https://docs.openwebui.com/getting-started/advanced-topics/api-endpoints/ which does not reference creation of users. Though I can query users https://mysite/api/v1/users when authenticated, though it does not work via curl command and authentication bearer.

Can you please advise what the proper procedure is? I would like to include creation/enabling/disabling users in an automation process.
Thanks!
S

**Comment by chrysllercs** [comment]
Hi,

I'm experiencing the same issue. I can query users using the endpoint https://mysite/api/v1/users when authenticated, but I'm unable to create, enable, or disable users as part of an automation process. I've tried using the curl command with authentication bearer, but it's not working.

Could you kindly provide guidance on the proper procedure for user creation and management?

Thank you!

**Comment by Fractiq** [comment]
I managed to create a user by sending a post request to https://{server}/api/v1/auths/signup with a body of 
`{ "body": { "email": "testemail@test.test", "name": "Jenny Rosen", "password": "randompassword", "profile_image_url": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAABkCAYAAABw4pVUAAAAAXNSR0IArs4c6QAABCZJREFUeF7tmktIFHEcx78zrutqL4kOJhSFQqeIuik9LmF1LoLOdYkgT0Udg6LHqSLr4qm6FPS6Rg9CMuiJGoISFiglFZgpsjs7sxOzm9aqK7/Zll8/6js35Tvz+83nM7//PNSZ6FwegpsZAg6FmHGRb4RCbPmgEGM+KIRCrBEw1g/vIRRijICxdjghFGKMgLF2OCEUYoyAsXY4IRRijICxdjghFGKMgLF2OCEUYoyAsXY4IRRijICxdjghFGKMgLF2OCEUYoyAsXY4IRRijICxdjghFGKMgLF2OCEUYoyAsXY4IRRijICxdlQmpHbXHVQ1bo156iHgpxFmJ5EbH4Q/dBf+u5v5n0ttVY1bkNp2BU7dynwkyma6j+T3i7slNx1HcsNhwE3md819G8DUrda4h4mdNyxk9rmEyH3/gGzPeWQHr897ohQi9F/ehMx/8Oiqz/Z1wHtzbk6AQsoRkvMRjHYjnBpdeG+3Gu6yJjhL1sBJLi3KRvumu9oRjDwo+j2FlCXEg9dzEd7r06K9nUWNqGk5g8TqnYBT9XOfEP77e0g/2k8hIoqzQkVLVi6ekOhQTvVipLZf/flg4BRu2JPDSD8+gODzy5lqnBChnT8VEpVJNO1BTctZODX1BSH+FLwXJ5Dt76QQoYeZWCWERPeTVNsNuEvXFo4bTVrvJXivTlHI3xAS1azb3Q23ft1M+ejxN9PVTiEmhHBC4mr4la/EkhU9ZdVsvgCndkXhwEEa3uuz8HovckLiqqmEkNTWDiSa9wKOW7ipT31C+slBBB+7KERbSHLjUVSvP5R//J3e/OH7SN/fV9TK7MfeuH0ulP93v2UJ30MSq9rgNrQgsXoH3GXNv70UAmFmDJnuo/CHblNIOVddJb9lIcjAe3sZ3suTc1rhhAjtVExIkEZ24Boyz47NW3mOkCADf+QhsMAn+1Kn4NQ1oKqhFXAThdcefn6fRhUizIwj+Pwc2b5LCD49LXkZ8NNJORMi/NobZieQ+9qDYKwfuS9vRJUoRIQJqMRjr6QUhUgogUKEmPIx/T/hCh9745zEdJYTIqTGJUsIihNSGtT/818nXLIWHBfeQ0rg4YTIl1m+GP4pK97U5QS5ZHHJiv9vQNLri+8hUlLMiQmoLFnibhjU+XRCznICnBA5K5UkhahglhehEDkrlSSFqGCWF6EQOSuVJIWoYJYXoRA5K5UkhahglhehEDkrlSSFqGCWF6EQOSuVJIWoYJYXoRA5K5UkhahglhehEDkrlSSFqGCWF6EQOSuVJIWoYJYXoRA5K5UkhahglhehEDkrlSSFqGCWF6EQOSuVJIWoYJYXoRA5K5UkhahglhehEDkrlSSFqGCWF6EQOSuVJIWoYJYXoRA5K5UkhahglhehEDkrlSSFqGCWF6EQOSuVJIWoYJYX+QEMffsSPik+RgAAAABJRU5ErkJggg==" }`

Even tho it does freeze up my openWebui server until something times out.. afterwards a user is made in openwebui
havent figured out why it freezes my server instance yet tho

EDIT:
I figured out what was freezing my openWebUI web instance, some webhook functions. i removed the URL from "Webhook-URL" under settings
